src/main/resources/content/k8s/do-not-allow-sharing-host-PID-namespace.json

```diff
@@ -1 +1,8 @@
-{"severity":"medium","rule":"$.spec.template.spec.hostPID is true","id":"4c5d00c1-8f60-40bc-9566-a5b4e019752a","enabled":true,"resourceType":"k8s","policy":"do not allow sharing host PID namespace"}
\ No newline at end of file
+{
+  "severity": "medium",
+  "rule": "$.spec.template.spec.hostPID is true",
+  "id": "4c5d00c1-8f60-40bc-9566-a5b4e019752a",
+  "enabled": true,
+  "resourceType": "k8s",
+  "policy": "do not allow sharing host PID namespace"
```

Comment:
description field missing here as well.
Also, the policy name is not sentence case.